packages/focusvisible/src/useFocusVisible.ts

```diff
@@ -28,27 +28,33 @@ const INPUT_TYPES_WHITE_LIST = {
   'datetime-local': true
 };
 
+export interface IUseFocusVisibleProps {
+  scope?: React.RefObject<HTMLElement | undefined>;
```

Comment:
This shouldn't be optional if we throw an `Error` when it is undefined.

Reply:
Is `RefObject<HTMLElement | undefined>` the default ref value?

I think `RefObject<HTMLElement | null>` would be more accurate

Reply:
Thanks for the suggestions. `scope` is now required and has type `RefObject<HTMLElement | null>`.